lib/apiservers/engine/backends/container_proxy.go

```diff
@@ -243,6 +244,41 @@ func (c *ContainerProxy) CreateContainerHandle(vc *viccontainer.VicContainer, co
 	return id, h, nil
 }
 
+// AddImageToContainer adds the specified image to a container, referenced by handle.
+// If an error is return, the returned handle should not be used.
+//
+// returns:
+//	modified handle
+func (c *ContainerProxy) AddImageToContainer(handle, deltaID, layerID, imageID string, config types.ContainerCreateConfig) (string, error) {
```

Comment:
Can we document this "deltaID" above.  I had to cross reference several location to figure out what this deltaID was.  Is there a reason we don't just use id as we've done in the other functions? I know id is not very precise, but now we have this one-off that refers to "deltaID".